src/gui/MainWindow.cc

```diff
@@ -97,6 +97,29 @@
 #include "QSettingsCached.h"
 #include <QSound>
 
+#ifdef ENABLE_PYTHON
+extern std::shared_ptr<AbstractNode> python_result_node;
```

Comment:
One comment for future PRs: With multiple MainWindows, this feels like a challenge. Should these be MainWindow members instead?

Reply:
Right, ideally we don't even need that if the processing switch between scad and python returns the root node.